addons/test_discuss_full/tests/test_performance_inbox.py

```diff
@@ -20,10 +20,11 @@ def test_fetch_rating_stats(self):
                 partner_ids=[self.user_employee.partner_id.id],
                 rating_value="4",
             )
+        self.env.registry.clear_cache()
```

Comment:
Why do we need to add this part?

Reply:
And to make both cases exactly the same, I have to put it before each assertion.